package zeke.exceptions;

public class CompletedTaskException {
}
